### PR TITLE
feat(markdown-models): add support for tables

### PR DESCRIPTION
**Pull request by TC5022**
Signed-off-by: TC5022 <tanyac5022002@gmail.com>

### Changes

- <ONE> This is a PR to introduce table extensions in the common mark and ciceromark models
- <TWO> This is required to complete the table transformations in other markdown formats

### Author Checklist
- [ ] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `master` from `fork:branchname`

### CODE REVIEW

File: src/markdown/commonmark@0.4.0.cto [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+concept TableHead extends Child{
+}
+
+concept TableBody extends Child{
+}
+
+concept TableRow extends Child{
+} 
+
+concept HeaderCell extends TableCell{
```
Comment (dselman) [inline src/markdown/commonmark@0.4.0.cto:127]: Extends Child?

**Review by dselman — CHANGES_REQUESTED** [review]
HeaderCell should extend Child

**Comment by TC5022** [comment]
> HeaderCell should extend Child

Fixed it @dselman

**Review by DianaLease — APPROVED** [review]
👍

**Review by dselman — APPROVED** [review]

**Pull request merged**